src/main/java/io/github/thebusybiscuit/slimefun4/integrations/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * This package holds classes which are related to integrations between Slimefun and Third-Party plugins.
+ */
+package io.github.thebusybiscuit.slimefun4.integrations;
```

Comment:
![image](https://user-images.githubusercontent.com/35664724/101798818-d50cd880-3b0b-11eb-8f68-286adfdcf4bc.png)
:)

Reply:
It's just a docs file, not really too important XD